packages/html/src/notification/tests/notification-angular.tsx

```diff
@@ -15,133 +15,92 @@ const styles = `
     }
 `;
 
-export default () =>(
+export default () => (
     <>
         <style>{styles}</style>
         <div id="test-area" className="k-d-grid k-grid-cols-4">
 
             <span className="k-colspan-all k-col-span-full">LTR</span>
-
-            <div>
-                <div className="k-notification-container">
-                    <Notification text="Default notification" />
-                </div>
-            </div>
-            <div>
-                <div className="k-notification-container">
-                    <Notification closable={true} text="Closable notification" actions={ <NotificationAction type="close" /> } />
-                </div>
-            </div>
             <div>
                 <div className="k-notification-container">
-                    <Notification themeColor="primary" text="Primary notification" />
+                    <IconNotification themeColor="info" text="Info notification" icon="info-circle" />
```

Comment:
In muy opinion this test should demonstrate only the types of notifications, but not some of the colors (as there is another test which demonstrates all colors)